### PR TITLE
Fixing line wrapping issue on lower res

### PR DESCRIPTION
**Pull request by leoman**

### CODE REVIEW

File: _sass/_typography.scss [hunk 1]
```diff
@@ -6,3 +6,5 @@
 
 /* Measure is limited to ~45 characters */
 .measure-narrow#{$suffix} { max-width: 20em; }
+
+.no-wrap#{$suffix} { white-space: nowrap; }
```
Comment (peduarte) [inline _sass/_typography.scss]: This works but I don't like forcing removing the white space. do you think we could solve this by simply increasing the width of the parent `.grid`? Currently its defined as 6 columns, would it work if it was 7?

https://github.com/fathomlondon/fathomlondon.github.io/blob/master/_includes/footer.html#L3

Comment (leoman) [reply _sass/_typography.scss]: I had a look, and it causes the same issue, the text still overflows. Interestingly, the issue only appears for me in chrome when resized to a small res, or using the dev tools, to see it in "ipad" mode. If we give the block more spacing, the social media icons will be forced onto their own line.

Comment (peduarte) [reply _sass/_typography.scss]: Interesting... to me this is not really a but. The fact that the text wraps is ok - just follows the nature of the content flow. It just looks weird.... maybe we should leave it then.

Comment (AndrewJDick) [reply _sass/_typography.scss]: Removed the margin-right for the contact card in the footer. 
That gives the text enough room to breathe before dropping down to the smaller breakpoints.

**Pull request merged**